packages/protocol/src/data/part-data.ts

```diff
@@ -39,7 +39,7 @@ export type ConditionData = ElementAlternative;
 export interface TaskData {
   task: WfTask;
   tasks: WfTask[];
-  persist: boolean;
+  persistOnStart: boolean;
```

Comment:
Somehow this was renamed, so no longer fitted the config...
I work on something to make this stuff more "typo" or schema safe.